README.md

```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Self-Introduction
+### Basic Information
+Hi, my name is Shuo Qiu, a third-year college student majoring in Sociology. Sociology is interesting to me because it can explain collective phenomenon that a study of only individuals cannot.  
```

Comment:
I recommend leaving a blank line between different Markdown elements, such as between headers and paragraph text. It avoids rendering issues in more complex documents